tests/integration/staking/keeper/liquid_stake_test.go

```diff
@@ -0,0 +1,274 @@
+package keeper_test
```

Comment:
Other keepers apart from the `StakingKeeper` are mocked in `x/staking/keeper/liquid_stake_test.go`,  so it is simpler to put these tests here since they require changing `x/bank` module states.

Reply:
gotchya, makes sense!